### ISSUE TITLE
ssh vs https url for git clone

### ISSUE THREAD

**Issue by michelleN** [fermyon/workshops#10]
In the setup section, does it make sense to use the ssh url rather the https url for cloning the workshops repo?

I'm not familiar with what the standard is for workshop content but I am very used to using the ssh url so just wanted to ask.

**Comment by kate-goldenring** [comment]
Good point @michelleN. I personally prefer ssh too. @radu-matei any reasoning behind using http?
